kornia/augmentation/utils/helpers.py

```diff
@@ -82,10 +128,30 @@ def _validate_input_dtype(input: torch.Tensor, accepted_dtypes: List) -> None:
         raise TypeError(f"Expected input of {accepted_dtypes}. Got {input.dtype}")
 
 
-def _validate_shape(shape: Union[Tuple, torch.Size], required_shapes: List[str] = ["BCHW"]) -> None:
+def _transform_output_shape(output: Union[torch.Tensor, Tuple[torch.Tensor, torch.Tensor]],
```

Comment:
Add some docs.

Also, I think this is a more augmentation-specific function if (tensor, tensor) involved. I would move this under augmentation module.

Reply:
as `utils` is under augmentation, `helpers` must be augmentation-specific already, no?

Reply:
Yes. My bad.